loadtest/control/actions.go

```diff
@@ -585,25 +585,28 @@ func Reload(u user.User) UserActionResponse {
 		return UserActionResponse{Err: NewUserError(err)}
 	}
 
-	prefs, _ := u.Store().Preferences()
+	prefs, err := u.Store().Preferences()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+
 	var userIds []string
-	var chanId string
-	// TODO: possibly remove this. Data should probably come from the store.
 	for _, p := range prefs {
 		switch {
-		case p.Name == model.PREFERENCE_NAME_LAST_CHANNEL:
-			chanId = p.Value
 		case p.Category == model.PREFERENCE_CATEGORY_DIRECT_CHANNEL_SHOW:
 			userIds = append(userIds, p.Name)
+		case p.Category == "group_channel_show":
```

Comment:
This is strange. I don't see this anywhere in the server codebase.

Reply:
I've actually added a note about this in the other action that makes use of it. 
There's no constant defined in model but it's actively used by the webapp client.

Reply:
Got it.